install.bat

```diff
@@ -0,0 +1,35 @@
+@echo off
+
+:: change current directory to the one where this script is located
+:: if executed as administrator it would change to system32 otherwise
+pushd %~dp0
+
+python --version 2>NUL
+if errorlevel 1 goto errorNoPython
+
+if exist %UserProfile%\.platformio\penv\Scripts\ (
+  echo PIO detected. Skip installation...
+) else (
+  python get-platformio.py
+)
+
+SET PATH=%PATH%;%UserProfile%\.platformio\penv\Scripts
+
+if exist board.txt (
+  set /p PIO_ENVIRONMENT=<board.txt
+) else (
+  echo board.txt not found
+  pause
+  exit
+)
+
+pio run -e %PIO_ENVIRONMENT% -t upload
+
+pause
+exit
+
+:errorNoPython
```

Comment:
Is it possible to install python through this script? The less complicated things a user has to do here the better.